www/src/components/landingPage/ClipboardSelect.tsx

```diff
@@ -4,19 +4,19 @@ import { Fragment, useState } from "react";
 
 const commands = [
   {
-    command: "create t3-app@latest",
+    command: "create-t3-app@latest",
```

Comment:
```
npm create t3-app@latest
yarn create t3-app
pnpm create t3-app@latest
bun create t3-app@latest
```

were the final commands given by @c-ehrlich 

![image](https://github.com/t3-oss/create-t3-app/assets/19780885/dec4418f-fdfc-4a0b-87fb-7ca65f874a31)

Reply:
Sorry about that! I must have missed that when I was going over the incorrect changes from the original PR. Will get that updated now.